### DISCUSSION TITLE
Python Programlama Ders 2. Değişkenler, deyimler ve cümleler | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#160] [Site]
# Python Programlama Ders 2. Değişkenler, deyimler ve cümleler | SonsuzUs Github

2.1 Değerler ve tipler

https://sonsuzus.github.io/posts/python-programlama-ders-02/

**Comment by ArvenAA** [comment]
çap = int(input("Dairenin çapını giriniz "))
r = çap/2
alan = 3.14*r**2
cevre = 2*3.14*r
print("Dairenin alanı", alan, "dairenin cevresi", cevre)

**Comment by yigidi** [comment]
TL= int(input("Kaç TL'niz var?: "))
Dolar= TL/28
Euro=TL/29
print(Dolar," Dolar ", Euro,"Euro alabileceksiniz")

**Comment by TheEmirAK** [comment]
R = int(input("çap ne"))
print ("çevre",3.14*R)
print ("alan",3.14*(R/2)*(R/2))

**Comment by yigidi** [comment]
import math
R=int(input("Çap giriniz: "))
r= R/2
pi= math.pi
cevre=2*pi*r
alan=pi* r**2

print("Dairenin alanı: ",alan," Çevresi: ", cevre)

**Comment by yigidi** [comment]
Saat= int(input("Saat kaç: "))
Süre=int(input("Alarmı kaç saat sonraya kuracaksınız: "))
Ç_saati=(Saat + Süre)%24
if Süre + Saat< 24 and Süre !=0:
    Ç_saati=Saat+ Süre
print("Alarm saat" ,Ç_saati, ":00 'da  çalacak")

**Comment by mervedam** [comment]
çap=int(input("dairenin çapını giriniz:"))
yç=çap/2
pi=3.14
print("dairenin alanı:",pi*yç*yç,"dairenin çevresi:",2*pi*yç)

**Comment by mervedam** [comment]
para=int(input("elinizdeki para miktarını giriniz (30'DAN FAZLA OLMALI) :"))
dolar=para/27,93
euro=para/29,51
print("(veriler 17.10.2023 saat 21 civarına aitir)\nParanızın dolar miktarı:",dolar,"\neuro miktarı:",euro)

**Comment by BerraDuruDemir** [comment]
çap=int(input("çap giriniz"))
print(çap)
r=int(çap/2)
alan=(3,14*(r**2))
çevre=(2*3,14*r)
print(alan)
print(çevre)

**Comment by BerraDuruDemir** [comment]
para=int(input("kaç tl niz var?"))
dolar=(para/27.93)
euro=(para/29.53)
print(dolar , "dolar", euro, "euro")

**Comment by BerraDuruDemir** [comment]
saat=int(input("saat kaç"))
saat2=int(input("kaç ssat sonra alarm çalsın"))
alarm=((saat+saat2)%24)
print("saat", alarm , ":00 uyanacaksın")

**Comment by ghost** [comment]
R = int(input ("Çemberin çapı nedir?"))
Pi = (3.141592653589796)
r = (R/2)
Alan = (Pi*r**2)
Çevre = (2*Pi*r)
print("Çemberin çevresi", Çevre)
print("Çemberin alanı", Alan)

**Comment by QLNH9491** [comment]
#Daire Sorusu
cap = int(input("Lütfen Dairenin Çapını Giriniz: "))
cap = cap/2
pi = 3.141592
print("Dairenin Çevresi: ",2*pi*cap)
print("Dairenin Alanı: ",pi*cap**2)
#Dolar-Avro Sorusu
dolar=27.93
avro=29.54
#ne kadar değişir bilmiyorum ama neyse
tl=int(input("Elinizdeki Türk Lirası? (Güncel değildir) "))
print("Dolar olarak elinizdeki Türk lirası: ",tl/dolar)
print("Avro olarak elinizdeki Türk Lirası: ",tl/avro)
#Normal Soru
saat = 14
dakika = 0 #opsiyonel
gun_saati = 24

eklenen_saat = 51
toplam_saat = saat + eklenen_saat
alarm_saat = toplam_saat % gun_saati
alarm_dakika = dakika

print("Alarm saat:",alarm_saat,":",alarm_dakika)
#Genel Soru
saat = int(input("Saati Giriniz: "))
dakika = int(input("Dakikayı Giriniz: "))
gun_saati = 24

eklenen_saat = int(input("Alarm kaç saat sonra olsun? "))
toplam_saat = saat + eklenen_saat
alarm_saat = toplam_saat % gun_saati
alarm_dakika = dakika

print("Alarm saat:",alarm_saat,":",alarm_dakika)

**Comment by Sadik1Dev** [comment]
```
cap=(input("çap: "))
cap = int(cap)

r=cap/2

pi=3.141592653589793

cevre = 2*pi*r

alan = pi*r*r

print("çapı",cap,"olan dairenin çevresi:",cevre,"alanı:",alan)
--------------------------------------------------------------------------------------------------
tl=input("kaç Türk Liranız var?")
tl=float(tl)

dolar = 28.01

euro = 29.49

dpara = tl/dolar

epara = tl/euro

print(tl,"Türk Lirası karşılığında",round(dpara,2),"kadar Dolar veya",round(epara,2),"kadar Euro alabilirsiniz")
--------------------------------------------------------------------------------------------------------------------------------------
gun=24

saat = input("24 saatlik formatta lütfen saati söyleyin: ")

alarm = input("kaç saat sonra alarmın çalmasını istiyorsunuz: ")

saat=int(saat)
alarm=int(alarm)

for i in range(1,alarm+1):
       
    saat+=1
    if saat==24:
        saat=0
       
print(saat,"saatinde alarmınız çalacak.")



```

**Comment by MnG-Official** [comment]
R = int(input ("Çemberin çapı nedir?"))
 Pi = (3.141592653589796) 
r = (R/2) 
Alan = (Pir**2) 
Çevre = (2Pi*r) 
print("Çemberin çevresi", Çevre) print("Çemberin alanı", Alan)

**Comment by Eceg25** [comment]
R = int(input("Dairenin çapını giriniz."))
r = R/2
pi = 3.14
alan = pi*r**2
cevre = 2*pi*r
print("dairenin alanı", alan, "dairenin çevresi",cevre)

**Comment by Eceg25** [comment]
tl = int(input("Kaç TL'niz var?"))
dolar = tl/27.97
euro = tl/29.58
print("Dolar", dolar, "Euro", euro)

**Comment by Eceg25** [comment]
saat = int(input("saat kaç?"))
eklenen_saat = int(input("alarm kaç saat sonrasına kurulu?"))
alarm = ((saat+eklenen_saat)%24)
print("alarm", alarm ,".00'da çalışır.")

**Comment by OmerB2** [comment]
```python
a =int(input("Hava kaç derece:"))
print("Hava",a*1.8+32,"fahrenayt.")
b = int(input("Kaç yaşındasınız:"))
print(2023-b,"yılında doğmuşsunuz"))
c = int(input("Dairenin yarıçapı ne kadar:"))
print("Dairenin çevresi:",2*3*c,"Dairenin alanı:",3*c**2)
d = int(input("Elinizde kaç Türk lirası var:"))
print("Elinizde",d*0.036,"dolar",d*0.034,"euronuz var")
```

**Comment by mehmet-11** [comment]
#Çapın alanı, yarıçapı ve çevresi
çap = int(input("Dairenin çapı:"))
yarıçap = çap/2
pi = 3.14
alan = pi * (yarıçap * yarıçap)
çevre = 2*pi*yarıçap
print("Çapı", çap, "cm olan dairenin alanı: ", alan, "cm2'dir")
print("Çapı", çap, "cm olan dairenin yarıçapı: ", yarıçap, "cm2'dir")
print("Çapı", çap, "cm olan dairenin çevresi: ", çevre, "cm2'dir")

#Türk lirasının dolar ve euroya çevrilmiş hali
Lira=int(input("Ne kadar Türk liranız var?"))
dolar=27.99
euro=29.62
print("Dolara çevrilmiş hali: ",Lira/dolar)
print("Euroya çevrilmiş hali: ",Lira/euro)

#Alarm zamanı
saat = int(input("Saat kaç: "))
dakika = int(input("Dakika kaç?: "))
günsaati = 24
ilavesaat = int(input("Alarm kaç saat sonra çalsın? "))
toplamsaat = saat + ilavesaat
alarmsaat = toplamsaat % günsaati
alarmdakika = dakika
print("Alarmınız saat:",alarmsaat,":",alarmdakika,"da çalacak. İyi uykular:)")
Mehmet Çınar Gül

**Comment by esmay123** [comment]
```
#dolar 28, euro 30

tl = int(input(" Kaç TL? :    "))

dolar = tl/28
euro = tl/30 

print("Dolar:",dolar)
print("Euro:",euro)

print("...................")

c = int(input("Hava Kaç Derece?:  "))
f = int(9*c/5 + 32)

print(c,"c")
print(f,"f")

print("...................")

x = int(input("Yaşın kaç?: "))
y = (2023-x)
print("Hesaplanan Doğum Tarihiniz", y)

print("...................")

pi = 3.14
cap = int(input("Dairenin Çapını Giriniz: "))
yaricap = cap/2
cevre = yaricap*pi*2
alan = pi*yaricap**2

print("Çevre: ",cevre)
print("Alan: ",alan)
```

**Comment by Burak2210** [comment]
Daire sorusu:
```
R = int(input("Dairenin çapını giriniz:"))
print("dairenin çevresi:", R * 3.14)
print("dairenin alanı:", (R / 2) ** 2 * 3.14)
```
Tl değeri *:(* sorusu:
```
tl = int(input("kaç TL'niz var:"))
print(int(tl/28), "dolarınız ve", int(tl/29.6), "euronuz var")
```
Saat sorusu:
```
saat = int(input("saat kaç:"))
gecen_sure = int(input("alarm kaç saate çalacak:"))
for i in range(1,(gecen_sure+1)):
    saat += 1
    if saat == 24:
        saat = 0
print(saat, "saatinde alarm çalacak")
```
Saat sorusuna örnek:
```
saat = 14
for gecen_sure in range(1,52):
    saat += 1
    if saat == 24:
        saat = 0
print(saat, "saatinde alarm çalacak")
```

**Comment by omersalih177** [comment]
çap=int(input("Dairenin çapı ne "))
r = çap/2
alan = 3.14 r**2
çevre = 3.14*r
print( "alan ","çevre")

**Comment by omersalih177** [comment]
saat = int(input( "Saat kaç? "))
zaman = int(input( "Alarmı hangi saate kuracaksınız? ")
X saati = ( saat+zaman)%15
if (zaman+saat)<15 and zaman ! 0
X saati = (zaman+saat)
print( "Alarm saat ",X saati,:09'da çalmaya başlayacak)

**Comment by omersalih177** [comment]
para = ("para miktarınızı giriniz")
dolar = para/28
euro = para/29
print( dolar,"dolar",euro,"euro")
